test/unit/utils/isSubDir.spec.js

```diff
@@ -0,0 +1,35 @@
+import { isSubDir } from 'lib/utils'
+
+describe('isSubDir', () => {
+  const tests = [
+    ['/foo', '/foo'],
+    ['/foo', '/bar'],
+    ['/foo', '/foobar'],
+    ['/foo', '/foo/bar'],
+    ['/foo', '/foo/../bar'],
+    ['/foo', '/foo/./bar'],
+    ['/bar/../foo', '/foo/bar'],
+    ['/foo', './bar'],
+    ['C:\\Foo', 'C:\\Foo\\Bar'],
```

Comment:
These tests will not run as expected on a non-windows machine (test is failing on linux on travis). You probably should make these tests tailored to the value of `os.platform()`